.github/workflows/build.yaml

```diff
@@ -121,17 +121,10 @@ jobs:
         with:
           fetch-depth: 0
 
-      - name: Setup .NET Core
-        uses: actions/setup-dotnet@v4
-        with:
-          dotnet-version: 8.0.*
-          #dotnet-quality: preview
-
```

Comment:
Why did you remove this?

Reply:
This is for the client build. There is no reason to install .NET for it.